FrontEnd/src/components/Navigation.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+
+import { Row, Col } from 'react-materialize';
+
+///section css
+import componentCSS from './Navigation.scss';
+
+class Navigation extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			pages: [
+				{
+					icon: 'book',
+					text: 'Library',
+					ref: '/index.html',
+					class: 'valign-wrapper',
+				},
+				{
+					icon: 'settings',
+					text: 'Settings',
+					ref: '/settings.html',
+					class: 'valign-wrapper',
+				},
+				{
+					icon: 'face',
+					text: 'Admin',
+					ref: '/admin.html',
+					class: 'valign-wrapper',
+				},
+			],
+		};
+
+		switch (this.props.isActive) {
```

Comment:
not sure this is the best way of doing this, also coz' this won't be able to switch between active elements just now, at least without changing the prop above